packages/flutter_test/lib/src/finders.dart

```diff
@@ -23,18 +23,42 @@ const CommonFinders find = CommonFinders._();
 class CommonFinders {
   const CommonFinders._();
 
-  /// Finds [Text] and [EditableText] widgets containing string equal to the
-  /// `text` argument.
+  /// Finds [Text], [EditableText], and optionally [RichText] widgets
+  /// containing string equal to the [text] argument.
+  ///
+  /// If [skipRichText] is disabled, [RichText] widgets (and therefore also
+  /// [Text] and [Text.rich] widgets) are matched by comparing the
+  /// [InlineSpan.toPlainText] with the given [text].
+  /// If [skipRichText] is enabled, all standalone [RichText] widgets are
+  /// ignored and [Text] matching shortcuts to [Text.data].
+  ///
+  /// For [EditableText] widgets, the [text] is always compared to the current
+  /// value of the [EditableText.controller].
+  ///
+  /// If the `skipOffstage` argument is true (the default), then this skips
+  /// nodes that are [Offstage] or that are from inactive [Route]s.
   ///
   /// ## Sample code
   ///
   /// ```dart
   /// expect(find.text('Back'), findsOneWidget);
   /// ```
   ///
-  /// If the `skipOffstage` argument is true (the default), then this skips
-  /// nodes that are [Offstage] or that are from inactive [Route]s.
-  Finder text(String text, { bool skipOffstage = true }) => _TextFinder(text, skipOffstage: skipOffstage);
+  /// This will match [Text], [Text.rich], and [EditableText] widgets that
+  /// contain the "Back" string.
+  ///
+  /// ```dart
+  /// expect(find.text('Close', skipRichText: false), findsOneWidget);
+  /// ```
+  ///
+  /// This will match [Text], [Text.rich], [EditableText], as well as standalone
+  /// [RichText] widgets that contain the "Close" string.
+  Finder text(
+    String text, {
+    bool skipRichText = true,
+    bool skipOffstage = true,
+  }) =>
+      _TextFinder(text, skipRichText: skipRichText, skipOffstage: skipOffstage);
```

Comment:
nit: keep this on one line or change it to a multi-line `{ return ... }` block

Reply:
Sounds good, I thought about how to do it but I did not see any blocks elsewhere, but it for sure is the better style choice.